# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/x11/xfree86/imst.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- xc/programs/Xserver/hw/xfree86/drivers/imstt/imstt_reg.h	2002-08-06 19:57:30.000000000 +0000
+++ xc/programs/Xserver/hw/xfree86/drivers/imstt/imstt_reg.h	2003-04-08 22:10:30.000000000 +0000
@@ -9,22 +9,11 @@
 
 static __inline__ void regw(unsigned long base_addr, unsigned long regindex, unsigned long regdata)
 {
-	asm volatile ("stwbrx %1,%2,%3; eieio"
-		      : "=m" (*(volatile unsigned *)(base_addr+regindex))
-		      : "r" (regdata), "b" (regindex), "r" (base_addr));
 }
 
 
 static __inline__ unsigned long regr(unsigned long base_addr, unsigned long regindex)
 {
-	register unsigned long val;
-
-	asm volatile ("lwbrx %0,%1,%2; eieio"
-		      : "=r" (val)
-		      : "b" (regindex), "r" (base_addr),
-		      "m" (*(volatile unsigned *)(base_addr+regindex)));
-
-	return (val);
 }
 
 
